libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -271,6 +271,15 @@ class MapboxNavigation(
      */
     val historyRecorder = MapboxHistoryRecorder(navigationOptions, logger)
 
+    /**
+     * Navigation native experimental API. Do not store a link to [Experimental] object, use it with
+     * `mapboxNavigation.experimental` because [MapboxNavigation] is responsible for it's lifecycle.
```

Comment:
```suggestion
     * Navigation native experimental API. Do not store a reference to [Experimental] object, always access the object via
     * `mapboxNavigation.experimental` because [MapboxNavigation] is responsible for it's lifecycle.
```